parallel-install/pkg/config/constants.go

```diff
@@ -0,0 +1,10 @@
+package config
```

Comment:
Do we really need a separate file for this? I would have expected these in the place where they are most meaningful.

Or at least in `config.go`, but not a dedicated file.

Reply:
I did not like putting it to `config.go` as these constants are not configurable in any way. But if you think that this place would be more appropriate for them I will move them!

Reply:
Well, constants are per definition not configurable. They are just immutable values that are just used in relation to the package.
Same as `http` package would have a `const STATUS_OK = 200`.